src/promptix/tools/hook_manager.py

```
@@ -0,0 +1,339 @@
+            return
+        
+        if not self.is_promptix_hook():
+            self.print_status("Active hook is not a Promptix hook", "error")
+            return
+        
+        self.print_status("Running hook test...", "info")
+        
+        try:
+            # Run the hook directly
+            result = subprocess.run([str(self.pre_commit_hook)], 
+                                  capture_output=True, text=True)
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)